apps/admin/frontend/src/api.ts

```diff
@@ -468,6 +470,24 @@ export const listPotentialElectionPackagesOnUsbDrive = {
   },
 } as const;
 
+export const getWriteInAdjudicationReportPreview = {
+  queryKey(): QueryKey {
+    return ['getWriteInAdjudicationReportPreview'];
+  },
+  useQuery() {
+    const apiClient = useApiClient();
+    return useQuery(
+      this.queryKey(),
+      () => apiClient.getWriteInAdjudicationReportPreview(),
+      {
+        cacheTime: 0,
+        staleTime: 0,
+        refetchOnWindowFocus: false,
+      }
```

Comment:
backend only has one preview at a time, so frontend only tracks a preview if it's mounting. the assumption here is that we only have one preview at a time. if we chose to change that, it would break (but could be easily fixed).